planetiler-core/src/main/java/com/onthegomap/planetiler/geo/TileCoord.java

```diff
@@ -38,11 +38,11 @@ public record TileCoord(int encoded, int x, int y, int z) implements Comparable<
     }
   }
 
-  private static int startIndexForZoom(int z) {
+  public static int startIndexForZoom(int z) {
     return ZOOM_START_INDEX[z];
   }
 
-  private static int zoomForIndex(int idx) {
+  public static int zoomForIndex(int idx) {
```

Comment:
I think we should just expose `hilbertEncode` and `hilbertDecode` here, that seems like more of a match for what consumers might need rather than exposing this internal api to get there.